common/app/views/fragments/audio/containers/flagshipContainer.scala.html

```diff
@@ -57,9 +58,24 @@
                     @getLatestEpisode(allCards).map { card =>
                         <div class="fc-item--type-media">
                             <div class="fc-podcast-container__episode">
-                                <div class="fc-podcast-container__episode-image">
-                                    <img src="@FlagshipFrontContainer.AlbumArtUrl"/>
-                                </div>
+                                @if(true){
+                                    @card.displayElement match {
+                                        case Some(InlineImage(image)) => {
+                                            <div class="fc-podcast-container__episode-image-story">
+                                                @itemImage(
+                                                    image,
+                                                    inlineImage = true,
+                                                    widthsByBreakpoint = Some(card.mediaWidthsByBreakpoint)
```

Comment:
@paperboyo are you suggesting changing what we send into `@itemImage` here? 
Selecting some slightly larger breakpoint to try and force a bigger image to be returned?